pages/glossary/embedded-image.md

```diff
@@ -15,7 +15,6 @@ An element presents an _embedded image_ when any of the following is true:
 - the element is an `svg` element having one or more `image` [descendants][] with a non-empty `href` [attribute value][]; or
 - the element has a [computed][] [`background-image`][background-image] CSS property with at least one [`image`][css-image] that is a [url reference][url-reference].
 
-
```

Comment:
Please undo this. Don't want an autocomplete fix to be in the file history of embedded-image.md.